app/src/main/java/org/kiwix/kiwixmobile/SearchActivity.java

```diff
@@ -99,6 +99,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     MenuItem searchMenuItem = menu.findItem(R.id.menu_search);
     MenuItemCompat.expandActionView(searchMenuItem);
     searchView = (SearchView) menu.findItem(R.id.menu_search).getActionView();
+    searchView.setMaxWidth(Integer.MAX_VALUE);
```

Comment:
Are you sure we want to set it to Integer.MAX_VALUE ? It would be a very high value. ```2^32-1```

Reply:
@srv-twry I set it to Integer.MAX_VALUE as it would adjust to phones with any screen size. The actual issue was that I have tested it with tablet and the close button is not aligned to right.